docs/index.md

```diff
@@ -1,54 +1,17 @@
-### What Are We Fighting For?
```

Comment:
It was meant as a double entendre - both as a melodramatic, moral hook "why do we fight?", and as a poke at objective functions.